ui/admin/tests/acceptance/storage-buckets/create-test.js

```diff
@@ -86,7 +90,7 @@ module('Acceptance | storage-buckets | create', function (hooks) {
     assert.strictEqual(getStorageBucketCount(), storageBucketCount + 1);
   });
 
-  test('users can create a new storage bucket with org scope', async function (assert) {
+  test('users can create a new storage bucket aws plugin type with org scope', async function (assert) {
```

Comment:
Do we need a similar test case for minIO?

Reply:
@DhariniJeeva 
You are right. I will add such test. As we discussed within stand up, and posted on the boundary-ui channel, [here message](https://hashicorp.slack.com/archives/C012GTH1C4E/p1713291713830339) for the sake of the release I will provide this test on a separate PR.